src/canvas/Canvas.ts

```diff
@@ -930,8 +930,8 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
         subTargets: targets,
         button,
         isClick,
-        pointer: this.getPointer(e),
-        absolutePointer: this.getPointer(e, true),
```

Comment:
an example of the bug

Reply:
should have been
```
        pointer: this.getPointer(e, true),
        absolutePointer: this.getPointer(e),
```

Reply:
@asturur bug 1.

Reply:
sorry but this is how is in other places too:
<img width="435" alt="image" src="https://github.com/fabricjs/fabric.js/assets/1194048/898d9f83-f44f-4a6b-b16f-b55c26b28548">

are you sure this is one instance of the bug?